storybook/src/Safei/components/Organisms/GeneralPost/ImagePollDefault.js

```diff
@@ -0,0 +1,33 @@
+import React from "react";
```

Comment:
- 2 Radio buttons are visible as shown in the screenshot below
- Upload button is not clickable

Reply:
that was fixed from the upload comp now